cable/grpc/logging_interceptor.go

```diff
@@ -0,0 +1,41 @@
+package cable_grpc
+
+import (
+	"context"
+
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+
+	fctx "github.com/foundation-go/foundation/context"
+)
+
+// LoggingUnaryInterceptor returns a gRPC unary interceptor that logs all incoming gRPC calls.
+// It logs the method details, request, response, and any potential errors.
+func LoggingUnaryInterceptor(log *logrus.Entry) func(context.Context, interface{}, *grpc.UnaryServerInfo, grpc.UnaryHandler) (interface{}, error) {
```

Comment:
`LoggingUnaryInterceptor` from grpc works with ctx data like correlationId which doesn't exist for requests from anycable-go that's why make separate interceptor